### DISCUSSION TITLE
Do you live in Egypt?

### DISCUSSION THREAD

**Discussion by abdo072** [N0-0NE-Dev/NoFasel#26] [General]
Hi , you are of course following the media and you must have known that the Egyptian government is putting site owners in jail , so if you live in Egypt i think you should delete all arabic series And movies that are owned by current Arab streaming services like watch it and shahid .

**Comment by abadawe-dev** [comment]
Hey @abdo072, thanks for your concern. I don't own own or operate any of the servers that have the Arabic content (or any other content) on them. I am not a "site owner", NoFasel scrapes all of its content from third-party sites, it is more like a web browser than anything else really.

**Reply by abdo072** [reply]
I understand that  but you know the government  may not differentiate between both actions

**Reply by abadawe-dev** [reply]
If they went after akwam and shut it down, the Arabic content on NoFasel will automatically stop working, I am not too worried about it.

**Reply by abdo072** [reply]
Yeah they did , but maybe as a precaution you can just hide the arabic series panel for a while until things settle down and decide what to do with them later

**Reply by abadawe-dev** [reply]
They didn't shut down akwam, they just blocked, you absolutely can still access it with a vpn. I am not too worried about all of this, if I ever get a DMCA claim or anything of that sort I will comply with it.
